serverlessworkflow/modules/ROOT/pages/testing-and-troubleshooting/kn-plugin-workflow-overview.adoc

```diff
@@ -114,27 +117,128 @@ kn workflow create --name my-project
 ----
 --
 
-. Add more extensions to the Quarkus project during its creation by using the `[-e|--extension]` flag as follows:
+[[proc-build-sw-project-kn-cli]]
+== Running a workflow project using Knative CLI
+
+After creating your workflow project, you can use the `run` command with `kn workflow` to build & run your workflow project in your current directory. 
+
+This will start a {product_name} docker image and map your local folder to this image.
+
+.Prerequisites
+* {product_name} plug-in for Knative CLI is installed.
++
+For more information about installing the plug-in, see <<proc-install-sw-plugin-kn-cli, Installing the {product_name} plug-in for Knative CLI>>.
+
+* A workflow project is created.
++
+For more information about creating a workflow project, see <<proc-create-sw-project-kn-cli, Creating workflow project using Knative CLI>>.
+* Minikube cluster is running locally.
+
+
+.Procedure
+. In Knative CLI, enter the following command to build and run your workflow project:
 +
 --
-.Create a project with `quarkus-jsonp` and `quarkus-smallrye-openapi` extensions
+.Run the project and start a local development image.
 [source,shell]
 ----
-kn workflow create --extension quarkus-jsonp,quarkus-smallrye-openapi
+kn workflow run
 ----
+--
+. Navigate to `localhost:8080/q/dev` to access the Development UI locally.
```

Comment:
By default, the `localhost:8080/q/dev` should pop up automatically in the default web browser once it is ready (unless `--open-dev-ui=false` is specified).

So maybe something like this would be more appropriate:
```suggestion
. Once the project is ready, the Development UI will be opened up in a browser automatically (on `localhost:8080/q/dev`).
```